satpy/scene.py

```diff
@@ -217,7 +217,6 @@ def end_time(self):
         """Return the end time of the file."""
         return self.attrs['end_time']
 
-    @property
     def missing_datasets(self):
```

Comment:
Why is this no longer a property? This is a backwards incompatibility. Not really the best thing for a refactor PR.

Reply:
I agree this shouldn't be in a refactor PR, I will remove it. The PR was hastily finished, and that needed more work obviously which hasn't been done yet.

However, the reason I have this here is that I believe it should be a function, not a property. I got properly confused when doing the refactoring because I wasn't expecting the value of `missing_dataset` to be changing as it did between `load` calls. Maybe that's just me, but it seemed much more appropriate to make this a function then.

Reply:
> because I wasn't expecting the value of missing_dataset to be changing as it did between load calls

Not sure I understand how making it a method stops it from changing between load calls. Also not sure why it wouldn't change between load calls. If you try to load something and it fails, it is missing, if you try to load something else and it fails, it is also missing.

Reply:
making it a method doesn't stop it from changing of course, but a method feels more dynamic as opposed to a property or attribute. Anyway this is changed back now.